### ISSUE TITLE
Changing settings when launching from command line

### ISSUE THREAD

**Issue by celsobessa** [ImageOptim/ImageOptim#310]
Hi, is it possible to pass arguments for changing settings like quality when using any one of the command line methods described on this page https://imageoptim.com/command-line.html ?

Something like: `/Applications/ImageOptim.app/Contents/MacOS/ImageOptim *.png --jpeg-quality=80` or `open -a ImageOptim . --lossless` ?

Thanks!

**Comment by kornelski** [comment]
Yeah, it'd be useful. Currently it can be done only by using `defaults write` commands to change global settings.

**Comment by celsobessa** [comment]
> Yeah, it'd be useful. Currently it can be done only by using `defaults write` commands to change global settings.

For my use case, it worked perfectly. A bash script that generates several lists of images on given folder, based on the size and type of the image (e.g. jpg less than 480 pixels, jpg between 480 and 1024pixels), and optimizes them with different settings for each list.
